layouts/docs.hbs

```diff
@@ -13,7 +13,7 @@
       <article>
         {{{ contents }}}
 
-        {{> prev-next-navigation }}
+        {{!> prev-next-navigation }}
```

Comment:
This would impact reference doc pages too. We probably shouldn't use this layout for guides and leave this line the way it was.

Reply:
What reference doc pages are you referring to?

Reply:
Oh, sorry. I misread. It'd probably be good to be able to keep the continuation feature though. Perhaps we should consider putting the guides in sub-folders and number-prefix them to keep some sort of order?

Reply:
I thought about that as well but decided against it (for now):
- The order and categories I chose are quite arbitrary, there's no "right" flow
- This would also change URLs and we'd need to setup redirects

This section will be changed anyway in the relaunch we're currently working on, so it's a temporary fix anyway.